### ISSUE TITLE
[COGNITION]: Promo Banner - Icons SHOULD be hidden from screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10461]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Our [Promo Banner component](https://design.va.gov/components/promo-banners) uses two icons on the left, that are appearing as empty tab stops for JAWS. These do not announce to users and could be confusing. Let's hide them using the updated snippets below.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Icons are not being announced to screen readers. JAWS in particular.

## Environment

* https://design.va.gov/components/promo-banners

## Possible Fixes (optional)

```diff
! PromoBanner.jsx#L28

<span className="fa-stack fa-lg">
- <i className="vads-u-color--white fa fa-circle fa-stack-2x" />
+ <i aria-hidden="true" className="vads-u-color--white fa fa-circle fa-stack-2x" role="presentation" />
- <i className={iconClasses} />
+ <i aria-hidden="true" className={iconClasses} role="presentation" />
</span>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #250](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/250) via [**ZenHub**](https://www.zenhub.com/)
